$NetBSD: patch-Makefile,v 1.3 2020/03/26 13:59:40 nia Exp $

- Remove OS specific conditinal with GNU make extension
- Misc options for pkgsrc builds.

--- Makefile.orig	2020-02-21 04:51:47.000000000 +0000
+++ Makefile
@@ -1,21 +1,13 @@
 TARGET		= nanotodon
 OBJS_TARGET	= nanotodon.o config.o messages.o
 
-CFLAGS = -g
-# optimization
-#CFLAGS+= -O2
 # Use $XDG_CONFIG_HOME or ~/.config dir to save config files
 CFLAGS+= -DSUPPORT_XDG_BASE_DIR
 
-LDFLAGS = 
-ifeq ($(shell uname -s),Darwin)
-	LIBS = -lcurl -ljson-c -lncurses -lpthread -lm
-else
-	LIBS = -lcurl -ljson-c -lncursesw -lpthread -lm
-endif
+LIBS = -lcurl -ljson-c -lncursesw -lpthread -lm
 
 # for pkgsrc build
-#CFLAGS+= -I/usr/pkg/include -I/usr/pkg/include/ncursesw -DNCURSES_WIDECHAR
-#LDFLAGS+= -L/usr/pkg/lib -Wl,-R/usr/pkg/lib
+CFLAGS+= -I/usr/pkg/include -I/usr/pkg/include/ncursesw -DNCURSES_WIDECHAR
+LDFLAGS+= -L/usr/pkg/lib -Wl,-R/usr/pkg/lib
 
 include Makefile.in
